gleam.php

```diff
@@ -1,21 +1,35 @@
 <?php
 /**
  * Plugin Name: Gleam
- * Plugin URI: https://gleam.io/
+ * Plugin URI: https://gleam.io/docs/faq#wordpress
  * Description: Gleam competition plugin. Requires PHP 5.3
  * Version: 1.0
```

Comment:
**Suggestion**
This is a signifigant change.  Perhaps we should update this version now?

Reply:
I think we should update it to 2.0 since we change how to javascript embedded?